### PR TITLE
Add conduct, contributing, and cla

### PR DESCRIPTION
**Pull request by auryn-macmillan**
This PR adds:
- CODE_OF_CONDUCT.md
- CONTRIBUTING.md
- CLA assistant workflow

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
+# Contributing to Gnosis Guild projects
+
+A big welcome and thank you for considering contributing to Gnosis Guild's open source projects! It’s people like you that make it a reality for users in our community.
+
+Reading and following these guidelines will help us make the contribution process easy and effective for everyone involved. It also communicates that you agree to respect the time of the developers managing and developing these open source projects. In return, we will reciprocate that respect by addressing your issue, assessing changes, and helping you finalize your pull requests.
```
Comment (samepant) [inline CONTRIBUTING.md:5]: issues should be plural i think. ('addressing your issues')

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+## Quicklinks
+
+- [Contributing to Gnosis Guild projects](#contributing-to-gnosis-guild-projects)
```
Comment (samepant) [inline CONTRIBUTING.md:9]: 'projects' should be capitalized

**Review by samepant — COMMENTED** [review]
Looks good to me, left a couple typo comments

File: CONTRIBUTING.md (hunk 1 above)
Comment (auryn-macmillan) [reply CONTRIBUTING.md:5]: Good catch.

File: CONTRIBUTING.md (hunk 2 above)
Comment (auryn-macmillan) [reply CONTRIBUTING.md:9]: Fixed!

**Pull request merged**